pkg/parser/cyclonedx_sbom.go

```diff
@@ -0,0 +1,119 @@
+package parser
+
+import (
+	"fmt"
+	// "errors"
+	"os"
+	"encoding/json"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/lockfile"
+	"github.com/safedep/vet/pkg/common/logger"
+)
+
+// Define the struct types that match the JSON structure
+type Bom struct {
```

Comment:
Can we leverage a standard library for CyloneDX spec instead of hand crafting the data structure? Example https://github.com/CycloneDX/cyclonedx-go